### PR TITLE
Add a switch to display only today events

### PR DESCRIPTION
**Pull request by matclab**

### CODE REVIEW

File: i3_agenda/__init__.py [hunk 1]
```diff
@@ -79,10 +83,11 @@ def main():
 
     closest = get_closest(events)
     if closest is None:
-        print("No events")
         return
 
     t = datetime.datetime.fromtimestamp(closest.unix_time)
+    if args.today and t.date() != datetime.now().date():
```
Comment (rosenpin) [inline i3_agenda/__init__.py:89]: Instead of doing it this way, it would probably be better and more correct to put this restriction in the API call.
Here:
``` python
  events_result = service.events().list(calendarId=id,
                                              timeMin=now,
                                              maxResults=max_results,
                                              singleEvents=True,
                                              orderBy='startTime').execute()
```
you could add a parameter `timeMax` so it will only query for events for this day.  
so you could pass to it today's date with 23:59  
this way, we will only get relevant events from the query phase

File: i3_agenda/__init__.py [hunk 2]
```diff
@@ -79,10 +83,11 @@ def main():
 
     closest = get_closest(events)
     if closest is None:
-        print("No events")
```
Comment (rosenpin) [inline i3_agenda/__init__.py:82]: why did you remove it?

File: i3_agenda/__init__.py (hunk 1 above)
Comment (matclab) [reply i3_agenda/__init__.py:89]: Nice. Thank you for the tip, I'll do that.

File: i3_agenda/__init__.py (hunk 2 above)
Comment (matclab) [reply i3_agenda/__init__.py:82]: Oups. It was not intended to be part of the PR. It is just a personal preference (My bar is quite full and I prefer to have no message than a "No events" indicator).
I can add another switch for this in another PR.
